src/parser/parser.yy

```diff
@@ -1349,7 +1349,28 @@ yield_columns
     ;
 
 yield_column
-    : expression {
+    : KW_VERTEX {
```

Comment:
Change to label instead of key word. And should rewrite to `$-._vertices`

Reply:
fetch sentence also use this, 
It is not appropriate to modify it to label

Reply:
Out column name/property name is case sensitive, the key word is not.

Reply:
The fetch should unify too.

Reply:
yield vertex will be banned, must use yield vertex as name

Reply:
It's not related to the case sensitive problem.